lib/widgets/infoView_widget.dart

```diff
@@ -30,15 +33,20 @@ class InfoView extends StatelessWidget {
                     style: Theme.of(context).textTheme.headline3,
                   )),
               ExpansionTile(
-                  title: Text("Gdzie znajdę najbliższą Mszę Świętą?",
+                  iconColor: AppColors.accentSelect,
+                  textColor: AppColors.accentSelect,
+                  leading: Icon(LineIcons.cross),
+                  title: Text("Najbliższa msza święta",
                       style: TextStyle(
-                          color: AppColors.primaryNormal,
+                          color: AppColors.primaryDark,
                           fontWeight: FontWeight.bold)),
+                  subtitle: Text("Gdzie i w jakich godzinach?"),
```

Comment:
Consider using consistent styles (perhaps refactoring Text function is the way to go)